qiskit_nature/algorithms/ground_state_solvers/minimum_eigensolver_factories/vqe_uvcc_factory.py

```diff
@@ -29,31 +30,66 @@
 from qiskit_nature.properties.second_quantization.vibrational import (
     VibrationalStructureDriverResult,
 )
+from qiskit_nature.deprecation import deprecate_property, deprecate_positional_arguments
 
 from .minimum_eigensolver_factory import MinimumEigensolverFactory
 from ...initial_points import InitialPoint, VSCFInitialPoint
 
+logger = logging.getLogger(__name__)
 
-class VQEUVCCFactory(MinimumEigensolverFactory):
-    """A factory to construct a VQE minimum eigensolver with UVCCSD ansatz wavefunction."""
 
+class VQEUVCCFactory(MinimumEigensolverFactory):
+    """Factory to construct a :class:`VQE` minimum eigensolver with :class:`UVCCSD` ansatz wavefunction.
+
+    Note: Changing the ansatz of the minimum_eigensolver of the factory won't affect
+        the ansatz of new solvers produced by the factory. On top of that the default
+        type for :class:`VQE` is not the same as the factory.
```

Comment:
> On top of that the default type for :class:`VQE` is not the same as the factory.

What does this intend to say. I am having a hard time with this bit.

The first part is really saying that any ansatz a user might directly set into VQE via the minumum_eigensolver, will be overwritten by the factory when producing a solver via get_solver, as the factory set's the ansatz via what is defined to the factory here. (Maybe this is clearer than the text that has `won't affect`?)

- - - 

And we also take over initial_point the same way too right as in it should be managed via the factory, similar to the ansatz, as the factory will overwrite whatever might be set in VQE,

Reply:
Changed.